ensime_shared/launcher.py

```diff
@@ -163,6 +163,13 @@ def _start_process(self, classpath):
         null = open(os.devnull, "r")
         java = os.path.join(self.config['java-home'], 'bin', 'java')
 
+        if(os.name == "nt"):
+            java += ".exe" #the java executable has the .exe extension in windows
```

Comment:
Not too happy with the '.exe' hack but I'm kinda ok with it since we manipulate the java binary path.
An idea that occurred to me is that we can only check that java-home exists (it is very unlikely that it exists but it doesn't contain the java bin) and then we can use same java bin path for all OSs, I understand that you can execute a binary in windows without specifying the '.exe' extension. Is that right?